library/src/test/kotlin/com/chuckerteam/chucker/api/ChuckerInterceptorTest.kt

```diff
@@ -630,4 +632,80 @@ internal class ChuckerInterceptorTest {
 
         assertThat(serverRequestContent.utf8()).isEqualTo("Hello, world!")
     }
+
+    @ParameterizedTest
+    @EnumSource(value = ClientFactory::class)
+    fun `chucker processes all requests when no skipEndpoints are provided`(factory: ClientFactory) {
+        val chuckerInterceptorWithoutSkipping = ChuckerInterceptorDelegate(
+            cacheDirectoryProvider = { tempDir }
+        )
+        val client = factory.create(chuckerInterceptorWithoutSkipping)
+        executeRequestForPath(client,"/","Response from /")
+        val transaction = chuckerInterceptorWithoutSkipping.expectTransaction()
+        assertThat(transaction.responseBody).isEqualTo("Response from /")
+
+        executeRequestForPath(client,"/skip/path","Response from /skip/path")
+        val secondTransaction = chuckerInterceptorWithoutSkipping.expectTransaction()
+        assertThat(secondTransaction.responseBody).isEqualTo("Response from /skip/path")
+    }
+
+    @ParameterizedTest
+    @EnumSource(value = ClientFactory::class)
+    fun `chucker skips requests when skipPaths are provided`(factory: ClientFactory){
+        val chuckerInterceptorWithoutSkipping = ChuckerInterceptorDelegate(
+            cacheDirectoryProvider = { tempDir },
+            skipPaths = listOf(
+                "",
+                "    ",
+                "bla",
+                "www.bla.c/skip/path",
+                "bla.c/skip/path",
+                "90",
+                "https://bla/",
+                "/skip/path",
+                "/skip//",
+                "http://localhost:8080/skip/path/ext"
+            )
+        )
+        val client = factory.create(chuckerInterceptorWithoutSkipping)
+
+        executeRequestForPath(client,"","Hello, world!")
+        chuckerInterceptorWithoutSkipping.expectNoTransactions()
+
+        executeRequestForPath(client,"    ","Hello, world!")
+        chuckerInterceptorWithoutSkipping.expectNoTransactions()
+
+        executeRequestForPath(client,"www.bla.c/skip/path","Hello, world!")
+        chuckerInterceptorWithoutSkipping.expectNoTransactions()
+
+        executeRequestForPath(client,"bla.c/skip/path","Hello, world!")
+        chuckerInterceptorWithoutSkipping.expectNoTransactions()
+
+        executeRequestForPath(client, "90","Hello, world!")
+        chuckerInterceptorWithoutSkipping.expectNoTransactions()
```

Comment:
This code looks like not formatted right (look at the space before/after commas)